apps/e2e/pages-storybook/src/app/shared/links/links.component.html

```diff
@@ -0,0 +1,14 @@
+<sky-link-list headingText="Links">
+  <sky-link-list-item>
+    <a href="#">Link 1</a>
+  </sky-link-list-item>
+  <sky-link-list-item>
+    <a href="#">Link 2</a>
+  </sky-link-list-item>
+  <sky-link-list-item>
+    <a href="#">Link 3</a>
+  </sky-link-list-item>
+  <sky-link-list-item>
+    <a href="#">Link 4</a>
```

Comment:
Buttons should also be supported with this work. Can we throw a `button` in one of these to confirm the styles are appropriate in the screenshot?

Reply:
Added a button example.